internal/app/organisation_store.go

```diff
@@ -93,6 +111,37 @@ func (s *organisationStore) CreateMemberInvite(ctx context.Context, organisation
 	return nil
 }
 
+func (s *organisationStore) CreateLPA(ctx context.Context, organisationID string) (*actor.DonorProvidedDetails, error) {
```

Comment:
I'm leaving this here for now. We could probably modify `donorStore.Create()` but I didn't want to have to think through the `LpaLink` implications until we need it for the dashboard.